packages/grafana-ui/src/components/ConfirmButton/DeleteButton.tsx

```diff
@@ -22,7 +26,25 @@ export const DeleteButton: FC<Props> = ({ size, disabled, onConfirm, 'aria-label
       disabled={disabled}
       onConfirm={onConfirm}
     >
-      <Button aria-label={ariaLabel} variant="destructive" icon="times" size={size || 'sm'} />
+      <Button
+        type={'button'}
+        aria-label={ariaLabel}
+        variant="secondary"
+        fill={'text'}
+        icon={'trash-alt'}
```

Comment:
nit: I think we can remove most of the curlies here:

```suggestion
        type="button"
        aria-label={ariaLabel}
        variant="secondary"
        fill="text"
        icon="trash-alt"
```